### PR TITLE
made base.html.twig, including header and footer

### PR DESCRIPTION
**Pull request by KevinRoig**

### CODE REVIEW

**Review by rickifox — APPROVED** [review]

File: templates/base.html.twig [hunk 1]
```diff
@@ -9,8 +9,40 @@
     {% endblock %}
 </head>
 <body>
-
-{% block body %}{% endblock %}
+    <header>
+        <div class="navbar">
```
Comment (RomainClair) [inline templates/base.html.twig:13]: div class navbar should probably be a nav

File: templates/base.html.twig [hunk 2]
```diff
@@ -9,8 +9,40 @@
...
+            </div>
+            <div class="passages">
+                Passages secrets
+            </div>
+            <div class="separator">
+                <img src="{{asset('build/images/vertical-line.png') }}" alt="">
+            </div>
+            <div class="contact">
+                Contact
+            </div>
```
Comment (RomainClair) [inline templates/base.html.twig:36]: And because this is a navigation menu : i.e. a list of links
It should be an html list of links not a div of divs

**Review by RomainClair — CHANGES_REQUESTED** [review]
You should use the semantic web tags to give meaning to your web pages
You should probably not commit empy files

**Comment by KevinRoig** [comment]
Changes will be made on "gallery" branch, i close this branch.

**Pull request closed without merging**